src/odemis/gui/main_xrc.py

```diff
@@ -623,7 +624,7 @@ def __init__(self, parent):
 
 def __init_resources():
     global __res
-    __res = xrc.XmlResource()
+    __res = xrc.EmptyXmlResource()
```

Comment:
This one shouldn't be changed. Maybe you're still using wxpython 3, instead of wxpython 4?